composer.json

```diff
@@ -12,15 +12,15 @@
         {"name": "Jonathan H. Wage", "email": "jonwage@gmail.com"}
     ],
     "require": {
-        "php": "^7.2",
-        "doctrine/annotations": "^1.2",
-        "doctrine/mongodb-odm": "^2.0.0",
+        "php": "^7.2 || ^8.0",
```

Comment:
Would it make sense to use `>=7.2` instead to keep it in line with Symfony's preferred versioning strategy?

Reply:
No, that's a terrible strategy